flaps/flaps_secretkeys.go

```diff
@@ -0,0 +1,159 @@
+package flaps
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+
+	fly "github.com/superfly/fly-go"
+)
+
+func (f *Client) sendRequestSecretkeys(ctx context.Context, method, endpoint string, in, out interface{}, qs url.Values, headers map[string][]string) error {
+	endpoint = fmt.Sprintf("/apps/%s/secretkeys%s", url.PathEscape(f.appName), endpoint)
+	if qs != nil {
+		endpoint += "?" + qs.Encode()
+	}
+	return f._sendRequest(ctx, method, endpoint, in, out, headers)
+}
+
+func (f *Client) ListSecretkeys(ctx context.Context, version *uint64) ([]fly.SecretKey, error) {
```

Comment:
I'd like to use either `SecretKey` or `Secretkey`. Not really sure that is one word or not tbh.